profiles/desc/guile_targets.desc

```diff
@@ -3,5 +3,6 @@
 
 # This file contains descriptions of GUILE_TARGETS USE_EXPAND flags.
 
+1-8 - Build only for GNU Guile 1.8
```

Comment:
hmm, this is a copy paste error on my part - "only" does not make sense for the non-single targets.  we can fix this after merging.  LGTM (but not to CI apparently)

Reply:
Hm the CI failure is because guile-1.8.8-r4 has SLOT="12/8", and guile-1.8.8-r100 has SLOT="1.8", and the `<guile-1.9` dep in texmacs-2.1-r1 matches both. Not sure what to do with the slots, as the two Guile revisions can't be installed together.

Reply:
probably making the slot explicit works

Reply:
That still leaves the fact that guile -r4 and -r100 are in different slots but can't be installed at once. Does it even matter in this case?

Reply:
No, that's fine/expected.

Reply:
Alright, added another fixup for this.